src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -2717,6 +2717,10 @@
     <value>© %1 Microsoft. All rights reserved.</value>
     <comment>{Locked="%1"}. Copyright statement, displayed on the About panel. %1 = the current year (4 digits)</comment>
   </data>
+  <data name="AboutFlyoutContribute" xml:space="preserve">
+    <value>Windows Calculator is made available under the MIT license. To learn more and contribute, check out the project on %HL%GitHub%HL%.</value>
```

Comment:
Changes look great! One last tweak -- can we update this string to "To learn how you can contribute to Windows Calculator, check out the project on GitHub"

Reply:
Updated

<img src="https://user-images.githubusercontent.com/10975439/101853682-01395100-3b2e-11eb-94d8-474bff26fb22.png" alt="About Page" width="400">

Reply:
Definitely my typo, but can we add back the period?

Reply:
Sure